invenio/templates/application.yaml

```diff
@@ -38,6 +38,16 @@ spec:
       labels:
         app: web
     spec:
+      initContainers:
+      - name: web-init
+        image: {{ .Values.web.image }}
+        command: [
+          '/bin/bash',
+          '-c',
+          'cp -R {{ .Values.web.statics.location }}/. /tmp/invenio/statics/']
+        volumeMounts:
+            - name: "invenio-statics"
```

Comment:
This might just be me but I don't like the word 'statics', just nitpicking here.

Reply:
renamed to assets